configs/rmq.properties

```diff
@@ -0,0 +1,11 @@
+rmq.host=d3-rmq
+rmq.ethIrohaExchange=iroha
+
+rmq.irohaCredential.accountId=withdrawal@notary
```

Comment:
But why `withdrawal`? Can we use a dedicated account for this purpose? Looks confusing to me.

Reply:
yeah, i forgot to add it